src/cpp/session/modules/clang/CodeCompletion.cpp

```diff
@@ -339,6 +375,12 @@ void discoverSystemIncludePaths(std::vector<std::string>* pIncludePaths)
    for (std::size_t i = 0, n = includePaths.size(); i < n; ++i)
       includePaths[i] = string_utils::trimWhitespace(includePaths[i]);
    
+#ifdef __APPLE__
+   // prepend system include paths for non-Apple clang
+   if (!isAppleClang)
```

Comment:
So the system include path we see at runtime is different if we are using R 4.0 versus < R 4.0? Want to make sure I understand (and that this works as expected if using pre R 4.0).